.github/workflows/_validate.yml

```diff
@@ -44,6 +44,9 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: Install playwright
+        run: pnpx playwright install --with-deps
```

Comment:
Looks like `pnpx` is deprecated in favor of `pnpm dlx`. Worked great for me locally where `pnpx` failed (I think it's been removed in my version of pnpm).

Reply:
I changed this to `pnpm dlx` and update the readme for it.